Saudox/resources/views/profissional/admin/editar_profissional.blade.php

```diff
@@ -26,30 +26,48 @@
                         <input value="{{ $profissional->rg }}" class="noscroll" onkeypress='validar_apenas_numeros(event)' placeholder="RG (apenas números)" type="number"  name="rg" id="rg">
+                        <label>CRM:</label><br>
                         <input value="{{ $profissional->numero_conselho }}" class="noscroll" onkeypress='validar_apenas_numeros(event)' placeholder="Número do CRM (apenas números)" type="number"  name="numero_conselho" id="numero_conselho">
+                        <label class="required">Telefone 1:</label><br>
                         <input value="{{ $profissional->telefone_1 }}" placeholder="Telefone" type="text" name="telefone_1" id="telefone_1">
+                        <label>Telefone 2:</label><br>
                         <input value="{{ $profissional->telefone_2 }}" placeholder="Segundo Telefone (opcional)" type="text" name="telefone_2" id="telefone_2">
+                        <label class="required">Email:</label><br>
                         <input value="{{ $profissional->email }}" placeholder="E-Mail" type="text" name="email" id="email">
+                        <label class="required">Nacionalidade:</label><br>
                         <input value="{{ $profissional->nacionalidade }}" placeholder="Nacionalidade" type="text" name="nacionalidade" id="nacionalidade">
+                        <label class="required">Descrição do conhecimento e experiência:</label><br>
                         <input value="{{ $profissional->descricao_de_conhecimento_e_experiencia }}" placeholder="Descrição do conhecimento e experiência" type="text" name="descricao_de_conhecimento_e_experiencia" id="descricao_de_conhecimento_e_experiencia">
 
                         <hr class="hr_form">
                         <h3>Endereço</h3>
+                        <label class="required">Estado:</label><br>
                         <input value="{{ $profissional->endereco->estado }}" placeholder="Estado" type="text" name="estado" id="estado">
+                        <label class="required">Cidade:</label><br>
                         <input value="{{ $profissional->endereco->cidade }}" placeholder="Cidade" type="text" name="cidade" id="cidade">
+                        <label class="required">Bairro:</label><br>
                         <input value="{{ $profissional->endereco->bairro }}" placeholder="Bairro" type="text" name="bairro" id="bairro">
+                        <label class="required">Rua:</label><br>
                         <input value="{{ $profissional->endereco->nome_rua }}" placeholder="Rua" type="text" name="nome_rua" id="nome_rua">
+                        <label class="required">Nº:</label><br>
                         <input value="{{ $profissional->endereco->numero_casa }}" class="noscroll" onkeypress='validar_apenas_numeros(event)' placeholder="Número" type="number" name="numero_casa" id="numero_casa">
+                        <label class="required">Complemento:</label><br>
```

Comment:
## Utilizar o required apenas quando for necessário. Observe a imagem em anexo, o campo "Não Nulo". Além disso, é importante observar os demais campos, além de endereço.
__Recomendo não utilizar o DBEAVER para verificar os campos não nulos, mas sim as tabelas na pasta database__
<br>
![image](https://user-images.githubusercontent.com/25617786/93834624-e3f19680-fc52-11ea-9632-71f56495420f.png)